example/android/app/build.gradle

```diff
@@ -28,7 +28,7 @@ apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 android {
     namespace 'com.argyle'
 
-    compileSdkVersion flutter.compileSdkVersion
```

Comment:
For some reason this was resolved as `33` even with latest stable Flutter version, so I set it explicitly.